src/components/items_list/ItemsList.js

```diff
@@ -86,17 +87,12 @@ class ItemsList extends React.Component {
                 {"active": this.props.index === index},
                 {"updated": entity.isUpdated || isBeingEdited}
             )}>
-
                 <ShowIf condition={Boolean(entity.id)}>
                     <ListItemIcon className="superscript-icon">
                         <CheckIcon/>
                     </ListItemIcon>
                 </ShowIf>
-
-                <ListItemIcon onClick={selectHandler}>
-                    <WidgetsIcon/>
-                </ListItemIcon>
-
+                {entity.isNonPeriodic ? <DeviceHubIcon/> : <WidgetsIcon/>}
```

Comment:
What happened to `<ListItemIcon onClick={selectHandler}>`? It even has an onClick handler, is it not used?

Reply:
Double checking.